### PR TITLE
Add accuracy

### LINKED ISSUE #295: Add accuracy
Accuracy is important, and adding `correct and rejected` and `correct and predicted` manually is not great.

### PR DESCRIPTION
**Pull request by gabegma**
Resolve #295 

## Description:
It adds a scrollbar in the performance table, but I think it is tolerable. 
Screen Shot 2022-11-21 at 4 02 30 PM
Screen Shot 2022-11-21 at 4 02 41 PM

## Checklist:
You should check all boxes before the PR is ready. If a box does not apply, check it to acknowledge it.

* [x] **ISSUE NUMBER.** You linked the issue number (Ex: Resolve #XXX).
* [x] **PRE-COMMIT.** You ran pre-commit on all commits, or else, you
  ran `pre-commit run --all-files` at the end.
* [x] **USER CHANGES.** The changes are added to CHANGELOG.md and the documentation, if they impact
  our users.
* [x] **DEV CHANGES.**
    * Update the documentation if this PR changes how to develop/launch on the app.
    * Update the `README` files and our wiki for any big design decisions, if relevant.
    * Add unit tests, docstrings, typing and comments for complex sections.

### CODE REVIEW

**Comment by gabegma** [comment]
Oups - I'll fix/edit the tests tomorrow.

**Review by JosephMarinier — APPROVED** [review]
Good idea! 👍

**Comment by JosephMarinier** [comment]
Nothing new, but we could look into redesigning the metrics cards on the Prediction Overview as they are now way too wide to fit on the target MacBook Pro 14" by default.
Screen Shot 2022-11-22 at 2 25 07 PM
It's not tooooo bad as they still fit all fit if you close the Controls pannel:
image
Or if you use a MacBook Pro 16":
Screen Shot 2022-11-22 at 2 26 01 PM

File: docs/docs/reference/custom-objects/metric.md [hunk 1]
```diff
@@ -33,8 +33,6 @@ with precision, recall and F1 below.
 
 === "Config Example"
 
-    By default, Azimuth loads Precision, Recall and F1 with `average=weighted`:
-
     ``` json
     {
       "metrics": {
```
[the remark this answers is not in the document]
Comment (gabegma) [reply docs/docs/reference/custom-objects/metric.md:38]: Sure. I had removed the text that mentioned that these were defaults value so I didn't need to add it here as well. But we can add it if you prefer.

**Comment by gabegma** [comment]
> Nothing new, but we could look into redesigning the metrics cards on the Prediction Overview as they are now way too wide to fit on the target MacBook Pro 14" by default.

Yeah, we could - at least the scrolling is quite nice when they don't fit.

**Pull request merged**